packages/wp-story-editor/src/components/documentPane/publish/publish.js

```diff
@@ -116,6 +123,24 @@ const LogoImg = styled.img`
   max-height: 96px;
 `;
 
+const RevisionsWrapper = styled.div`
+  width: 100%;
+  margin-bottom: 20px;
+  margin-top: 4px;
```

Comment:
Minor styling suggestion

```suggestion
  margin-bottom: 16px;
  margin-top: 20px;
```